et_ee.lang

```diff
@@ -0,0 +1,402 @@
+gui.xaero_minimap_shape_circle=ring
+gui.xaero_slopes_legacy=Pärand
+gui.xaero_slopes_default=Vaikimisi
+gui.xaero_names_list=Mängijate loend
+gui.xaero_names_always=Alati
+gui.xaero_names_off=Nimedeta
+gui.xaero_entity_radar_type=Tüüp:
+gui.xaero_entity_radar_color=Värv:
+gui.xaero_entity_radar_visiblity=Nähtavus:
+gui.xaero_entity_radar_names=Nimed:
+gui.xaero_entity_radar_icons=Ikoonid/pead:
+gui.xaero_box_entity_radar_names=Kuva olemite nimesid täppide või ikoonide/peade all. \n \n Valik "mängijate loend" - kuva vaid siis, kui klahviotseteed "kuva mängijad" (vaikimisi tabeldusklahv) on vajutatud.
+gui.xaero_light_overlay_status=Valguse ülekatte kasutamine
+gui.xaero_light_overlay=Valguse ülekate
+gui.xaero_light_overlay_type=Valguse ülekate
+gui.xaero_light_overlay_color=Värv
+gui.xaero_light_overlay_max_light=Maksimaalne valgus
+gui.xaero_light_overlay_min_light=Minimaalne valgus
+gui.xaero_toggle_light_overlay=Lülita valguse ülekatet
+gui.xaero_box_light_overlay_type=Selle ülekatte kasutamine võib mõjutada mängu jõudlust negatiivselt, kuna hetkel see ei kasuta World Map modi kamakaid ning seega suurendab minikaardi uuendussagedust. Ära jäta seda funktsiooni pikaks ajaks lubatuks.
+gui.xaero_day=Päev
+gui.xaero_tamed=Kodustatud elukad
+gui.xaero_untamed_color=Kodustamata värv
+gui.xaero_icons_list=Mängijate loend
+gui.xaero_icons_always=Alati
+gui.xaero_icons_off=Ikoonideta
+gui.xaero_box_entity_radar_icons=Kuva värvitud täppide asemel olemite ikoone. \n \n Valik "mängijate loend" - kuva vaid siis, kui klahviotseteed "kuva mängijad" (vaikimisi tabeldusklahv) on vajutatud.
+gui.xaero_dot_name_scale=Täpi nime suurus
+gui.xaero_entity_heads_scale=Olemi ikooni suurus
+gui.xaero_box_minimap_size=Minikaardi raami suurus, mis mõjutab nähtava osa hulka. Suuruse üle 180 korral sunnitakse ruudukujuline minikaart mittepööravasse režiimi (valik "Minikaardi põhjasuuna lukk"). \n \n "Automaatse" valiku saamiseks liiguta täiesti vasakule. \n \n Suurused vanemates modi versioonides: \n Pisike - 57 \n Väike - 85 \n Keskmine - 113 \n Suur - 169
+gui.xaero_auto_clamp_depth=automaatne
+gui.xaero_waypoints_clamp_depth=Klambri sügavus
+gui.xaero_waypoints_distance_scale=Teepunkti kauguse teksti suurus
+gui.xaero_waypoints_name_scale=Teepunkti nime teksti suurus
+gui.xaero_box_waypoints_clamp_depth=Ekraani sügavus, millest alates mängusisesed teepunktid ei lähe nendest kaugenedes enam väiksemaks. \n Kasuta seda, kui soovid teha teepunkte, mis on kaugelt vaadates ekraanil suuremad/väiksemad. Et mõjutada teepunktide suurust, kui oled neile lähedal, kasuta seadet "mängusisese teepunkti suurus". \n \n "Automaatse" valiku saamiseks liiguta täiesti vasakule. \n Automaatne valik proovib anda sulle parima väärtuse suurus tüüpide vaikeväärtuste jaoks, võttes arvesse sinu vaatevälja ja ekraani kõrgust. \n \n Vaikeväärtused vanemates modi versioonides: \n Klambri sügavus: 12 \n Mängusisene suurus: 2,5 \n Nime teksti suurus: 0,5 \n Kauguse teksti suurus: 0,8
```

Comment:
"Use this, if you want to make waypoints, that are bigger/smaller on the screen when looked from afar."

Reply:
That doesn't sound right to me.

Reply:
Could you elaborate on the original meaning then?

Reply:
It should say something like "Use this, if you want to make waypoints that are far away bigger or smaller on the screen."

I don't know if you're reading what you have written wrong because it really doesn't make sense to me. It sounds like the setting allows you to make bigger/smaller waypoints, when it actually also affects existing ones.

Reply:
Think I got it now

Reply:
Seems good!